Helper/SnapchatHelper.php

```diff
@@ -483,6 +487,8 @@ private function getRequest(
                     $this->refreshToken();
                     sleep(self::$betweenOpSleep);
                 } else {
+                    // We're not sure to handle this, nop.
+                    $done = true;
                     sleep(self::$betweenOpSleep);
```

Comment:
Do we need to sleep here?

Reply:
I'm not sure, is this sleep here so we don't get throttled by Snapchat? @heathdutton

Reply:
Because when not sure what happened and we're hitting the API a lot, I'd say it's better to sleep here just in case.